packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -631,9 +631,7 @@ export async function createHotReloaderTurbopack(
     getNextErrorFeedbackMiddleware(opts.telemetry),
   ]
 
-  const versionInfoPromise = getVersionInfo(
-    isTestMode || opts.telemetry.isEnabled
```

Comment:
wondering if we had this disabled in test mode due to network flakes? but lgtm if it isn't causing problems.

Reply:
Maybe we should just move these checks lower. The installed version should always be displayed but whether we check for latest/canary should continue to be tied to telemetry settings.